packages/grid/_modules_/grid/locales/deDE.ts

```diff
@@ -7,7 +7,7 @@ const deDEGrid: Partial<GridLocaleText> = {
   rootGridLabel: 'grid',
   noRowsLabel: 'Keine Einträge',
   // noResultsOverlayLabel: 'No results found.',
```

Comment:
@sebastianfrey thanks for the fix. Can I ask you to also translate this key and uncomment it 😇

Reply:
There you go 9888f46